custom/icds_reports/utils/aggregation_helpers/aww_incentive.py

```diff
@@ -47,7 +47,8 @@ def aggregation_query(self):
           WHERE awcm.month = %(month)s AND awcm.state_id = %(state_id)s and awcm.aggregation_level=5
           GROUP BY awcm.awc_id, awcm.block_id, awcm.state_name, awcm.district_name,
                    awcm.block_name, awcm.supervisor_name, awcm.awc_name, awcm.aww_name,
-                   awcm.contact_phone_number
+                   awcm.contact_phone_number, awcm.wer_weighed, awcm.wer_eligible,
+                   awcm.awc_days_open
```

Comment:
why do you need to group by these extra columns? shouldn't grouping by location ids be sufficient to ensure that there's only one record for each awc?

Reply:
all columns must be in the group by or the aggregate, and since there are guaranteed to only be one row for each of those it was safe to add them to the group by (issue with aggregating from one table and having one for the ohter). I plan to refactor this with either a subselect or a cte in the next set of changes, but wanted minimum working query to get out tonight so i can have data in the report tomorrow